packages/__docs__/src/CodeSandboxButton/index.tsx

```diff
@@ -73,14 +73,15 @@ import 'moment/min/locales'
       ', '
     )}} from "@instructure/ui"\n\n`
 
-    const codeLines = this.props.code.split('\n')
-    const shouldRender = !codeLines[codeLines.length - 1]
-      .split(' ')
-      .join('')
-      .includes('render(<')
-    const codeBlock = shouldRender
-      ? `render(${this.props.code})`
-      : this.props.code
+    const cleanedCode = this.props.code.replace(/\s+/g, '')
+    const lastRenderIndex = cleanedCode.lastIndexOf('render(<')
+    const endsWithRenderStatement =
+      lastRenderIndex !== -1 && cleanedCode.endsWith('>)')
+
+    const codeBlock = endsWithRenderStatement
+      ? this.props.code
+      : `render(${this.props.code})`
+
```

Comment:
The old code split the code into lines and checked if the last line included 'render(<'. However, sometimes the last line was just a ')' despite the code ending in a render function. This resulted in multiple render functions in CodeSandbox. Now, all newline characters and extra spaces are removed from the code, and it is checked whether the code actually ends with render(<...>) like 'render(<Example />)' or 'render(<View><Component /></View>)